app/models/channel.rb

```diff
@@ -335,6 +337,22 @@ def notify_slack
     ).perform
   end
 
+  # Needed for bitFlyer, but can likely be used for Uphold too.
+  def create_deposit_id
```

Comment:
I'd rather we make this a JSONB column that says `deposit_setting` and one of the values can be `deposit_id`. Otherwise, we have an existing pattern mentioned in `app/models/uphold_connection_for_channel.rb` that I prefer we copy.